### PR TITLE
#300 load more information on profile page

### PR DESCRIPTION
**Pull request by Lvyshnevska**
Read More component has been updated with Ant Design ellipsis according to design and clarified requirements.

### CODE REVIEW

File: FrontEnd/src/components/ProfileDetail/DetailedInfo/ReadMore.jsx [hunk 1]
```diff
@@ -1,28 +1,42 @@
...
-    text ? (
-    <p className={classes['read-more']}>
-        {displayText}
-        {text.length > maxTextLength && (
-          <span onClick={toggleReadMore} className={classes['read-or-hide']}>
-            {!readMore ? 'читати далі' : 'приховати'}
-          </span>
-      )}
-    </p>) : null
+    <>
```
Comment (popovycholeg) [inline FrontEnd/src/components/ProfileDetail/DetailedInfo/ReadMore.jsx:23]: No need to add a fragment if there is only one element

File: FrontEnd/src/components/ProfileDetail/DetailedInfo/ReadMore.jsx [hunk 2]
```diff
@@ -1,28 +1,42 @@
...
+      <Paragraph
+        className={classes['read-more']}
+        onClick={toggleReadMore}
+        ellipsis={
+          ellipsis
+            ? {
+                rows: 6,
+                expandable: true,
+                symbol: ellipsisSymbol,
+              }
```
Comment (popovycholeg) [inline FrontEnd/src/components/ProfileDetail/DetailedInfo/ReadMore.jsx:33]: Could we move it to constant with a meaningful name?

Comment (Lvyshnevska) [reply FrontEnd/src/components/ProfileDetail/DetailedInfo/ReadMore.jsx:33]: Should we do it if it is Ant Design component and its structure?

**Review by popovycholeg — APPROVED** [review]
LGTM! Left couple comments

File: FrontEnd/src/components/ProfileDetail/ProfileDetailPage.jsx [hunk 3]
```diff
@@ -10,15 +10,15 @@ import BannerImage from './BannerImage';
...
 function ProfileDetailPage({ isAuthorized }) {
+  const authToken = localStorage.getItem('Token');
   const { id } = useParams();
   const urlProfile = `${process.env.REACT_APP_BASE_API_URL}/api/profiles/${id}`;
 
   async function fetcher(url) {
     const headers = {
       'Content-Type': 'application/json',
     };
-    if (isAuthorized) {
```
[the remark this answers is not in the document]
Comment (Lvyshnevska) [reply FrontEnd/src/components/ProfileDetail/ProfileDetailPage.jsx:20]: It's passed to child components

File: FrontEnd/src/components/ProfileDetail/DetailedInfo/ReadMore.jsx (hunk 1 above)
Comment (Lvyshnevska) [reply FrontEnd/src/components/ProfileDetail/DetailedInfo/ReadMore.jsx]: removed

**Review by Shushunya — APPROVED** [review]

File: FrontEnd/src/components/ProfileDetail/DetailedInfo/ReadMore.jsx (hunk 2 above)
Comment (Lvyshnevska) [reply FrontEnd/src/components/ProfileDetail/DetailedInfo/ReadMore.jsx]: Done

**Pull request merged**